packages/firebase_database/lib/src/database_reference.dart

```diff
@@ -193,6 +193,14 @@ class ServerValue {
   static const Map<String, String> timestamp = <String, String>{
     '.sv': 'timestamp'
   };
+
+  /// Returns a placeholder value that can be used to atomically increment the
+  /// current database value by the provided delta.
+  static Map<dynamic, dynamic> increment(int delta) {
```

Comment:
This return type can be `Map<String, dynamic>`, same for one below.

Reply:
Can update this in a follow-up PR I think, be good to get this shipped